backfill.sh

```diff
@@ -132,7 +132,7 @@ set +o nounset; . install.sh; set -o nounset;
 
         if [ ! -f $xmlpath ] && [ ! -f $xmlunpubpath ]; then
             # lax doesn't know where the remote location and it's not on the fs
-            if [ $remotepath = "no-location-stored" ]; then
+            if [ "$remotepath" == "no-location-stored\r" ]; then
```

Comment:
hrm - mine was returning `\r` and breaking with the error you reported. it's possible the PR on lax (which is now using a csvwriter) is adding the \r 

in fact, that makes all kinds of sense. I think the csvwriter defaults to excel-friendly csv.